docs-src/docs/guides/terraform_beyond_layer0.md

```diff
@@ -150,7 +150,180 @@ This is then used to populate the template fields in our [Dockerrun.aws.json](ht
+  Pre-existing state was found in "local" while migrating to "consul". An existing
+  non-empty state exists in "consul". The two states have been saved to temporary
+  files that will be removed after responding to this query.
+  
+  One ("local"): /var/folders/n0/19h9crxn2v75g70bl0txdj1wm05615/T/terraform529524247/1-local.tfstate
+  Two ("consul"): /var/folders/n0/19h9crxn2v75g70bl0txdj1wm05615/T/terraform529524247/2-consul.tfstate
+  
+  Do you want to copy the state from "local" to "consul"? Enter "yes" to copy
+  and "no" to start with the existing state in "consul".
+
+  Enter a value: 
+
+```
+
+Go ahead and enter: `yes`.
+
+```
+
+Successfully configured the backend "consul"! Terraform will automatically
+use this backend unless the backend configuration changes.
+
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
+any changes that are required for your infrastructure. All Terraform commands
+should now work.
+
+If you ever set or change modules or backend configuration for Terraform,
+rerun this command to reinitialize your environment. If you forget, other
+commands will detect it and remind you to do so if necessary.
+
+```
+
+### What's happening
+As you are configuring a backend for the first time, Terraform will give you an option to migrate your state to the new backend. From now on, any further changes to your infrastructure made by Terraform will result in the remote state file being updated. For more information about see (Terraform backends)[https://www.terraform.io/docs/backends/index.html].
+
+A new team member can use the `layer0.tf` from their own machine without obtaining a copy of the state file `terraform.tfstate` as the configuration points to a remote backend where the state file will be retrieved from.
+
+The command `terraform init` must be called:
+ * on any new environment that configures a backend
+ * on any change of the backend configuration (including type of backend)
+ * on removing backend configuration completely
+
+## Part 5: Scaling a Layer0 service
+The workflow to make changes to your infrastructure generally involves updating your Terraform configuration file followed by a `terraform plan` and `terraform apply`.
+
+### Update the Terraform configuration
+Open the file `layer0.tf` in a text editor and make the change to add a `scale` property with a value of `3` to the `layer0_service` section. For more information about the `scale` property, see [Layer0 Terraform Plugin](http://layer0.ims.io/reference/terraform-plugin/#service) documenation. The end result should look like the below:
+
+
+layer0.tf
+
+```
+# Create a service named "guestbook"
+resource "layer0_service" "guestbook" {
+  name          = "guestbook"
+  environment   = "${layer0_environment.demo.id}"
+  deploy        = "${layer0_deploy.guestbook.id}"
+  load_balancer = "${layer0_load_balancer.guestbook.id}"
+  scale         = 3
```

Comment:
[nit] if users add `wait = true` into the service configuration, it will behave the same as `l0 service [create|update|scale] --wait ...`